sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -260,32 +268,63 @@ def _delay_failure(start_time: datetime.datetime) -> None:
 
 def _get_headers(request_type, **kwargs) -> str:
     headers = kwargs.pop("headers", {})
-    if os.environ.get(REQUEST_TRACING_DISABLED_ENVIRONMENT_VARIABLE, default="").lower() != "true":
-        correlation_context = "RequestType=" + request_type
-        if (
-            "keyvault_credential" in kwargs
-            or "keyvault_client_configs" in kwargs
-            or "secret_resolver" in kwargs
-            or kwargs.pop("uses_key_vault", False)
-        ):
-            correlation_context += ",UsesKeyVault"
-        host_type = ""
-        if AzureFunctionEnvironmentVariable in os.environ:
-            host_type = "AzureFunction"
-        elif AzureWebAppEnvironmentVariable in os.environ:
-            host_type = "AzureWebApp"
-        elif ContainerAppEnvironmentVariable in os.environ:
-            host_type = "ContainerApp"
-        elif KubernetesEnvironmentVariable in os.environ:
-            host_type = "Kubernetes"
-        elif ServiceFabricEnvironmentVariable in os.environ:
-            host_type = "ServiceFabric"
-        if host_type:
-            correlation_context += ",Host=" + host_type
-        headers["Correlation-Context"] = correlation_context
+    if os.environ.get(REQUEST_TRACING_DISABLED_ENVIRONMENT_VARIABLE, default="").lower() == "true":
+        return headers
+    correlation_context = "RequestType=" + request_type
+
```

Comment:
Do we have a way to identify dev environments in python apps?

Reply:
Not one that I can think of.